packages/frontend/src/Layout/Nav/Nav.styled.ts

```diff
@@ -113,24 +133,50 @@ export const NavItem = styled.li`
   text-align: center;
 `;
 
+export const DesktopNavItems = styled.ul`
+  display: none;
+  @media ${Breakpoint.Tablet} {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 0;
+    margin-left: 60px;
+    list-style: none;
+  }
+`;
+
 export const StyledLink = styled(Link)<LinkProps>`
-  display: block;
+  display: ${({ $isDesktop }) => ($isDesktop ? "none" : "block")};
   font-size: ${FontSize.Medium};
   color: ${({ $isActive }) => ($isActive ? `${Color.White}` : `${rgba(`${Color.White}`, 0.4)}`)};
   padding: 8px 0;
-  transition: 0.2s;
+  transition: color 0.2s;
   :hover {
     color: ${Color.White};
-    font-size: ${FontSize.Medium};
+  }
+  :nth-child(2) {
+    margin-right: 14px;
+  }
+
+  @media ${Breakpoint.Tablet} {
+    display: block;
+    color: ${({ $isActive }) => ($isActive ? `${Color.Red}` : Color.Grey)};
+    font-weight: 500;
+    font-size: ${FontSize.SmallS};
+    padding: 15px;
+
+    :hover {
+      color: ${Color.Red};
+    }
   }
 `;
 
 export const SocialsWrapper = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: center;
-  padding: 0 70px;
+  width: 100px;
   margin-top: 40px;
+  align-self: center;
```

Comment:
ta właściwośc nie jest przypadkie od grida

Reply:
mieli ją dodawać do flexa jak coś to daj znać jak z tym jest

Reply:
Według dokumentacji to można stosować we flexie i gridzie 👍 🐸